csv.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const baseUrl = 'https://raw.githubusercontent.com/TheThingsNetwork/lorawan-devices/master/vendor';
+
+// Function to extract data from device.yaml and create a CSV line
+const extractData = (filePath, vendor) => {
+    try {
+        const fileContents = fs.readFileSync(filePath, 'utf8');
+        const data = yaml.load(fileContents);
+        if (data && data.name && data.description) { //only read files with these in its content
+            const name = data.name;
+            const description = data.description.replace(/"/g, "'");
+            const sensors = Array.isArray(data.sensors) ? `"${data.sensors.join(', ')}"` : '';
+            const imageUrl = data.photos?.main ? `"${baseUrl}/${vendor}/${data.photos.main}"` : '';
```

Comment:
At some point we have to sync this to a cloud bucket (like S3) with content delivery network (like CloudFront) and resize images in certain fixed dimensions. Some of the photos here are very big, as they should be because that allows us to do high res stuff with them.

Reply:
So.. is there anything you want me to do with it? Sorry if i'm not getting the request clear.

Reply:
That depends on what will happen with these photos. We shouldn't render them as-is: they're too big (and they should be that big) and GitHub raw's URL is too unreliable to have nothing between the user's browser and this URL.

I don't have a request for change for this PR, but it may need to be implemented later and then this script will need the update.

Reply:
I am also not sure myself how these will be rendered, but i found the raw github url to be the best for now.
Best to hear from @benolayinka if it's the right method.

Reply:
for now it's fine as you have it, at least for testing. in the future we will need to resize images and link to the web-friendly size.